examples/basic_cxx_qt_qml/src/data.rs

```diff
@@ -27,9 +27,8 @@ mod my_data {
 
     impl RustObj {
         #[invokable]
-        fn as_json_str(&self, cpp: Pin<&mut CppObj>) -> String {
-            let wrapper = CppObjWrapper::new(cpp);
-            let data = Data::from(&wrapper);
+        fn as_json_str(&self, cpp: &mut CppObj) -> String {
+            let data = Data::from(&*cpp);
```

Comment:
At the moment we only implement `impl<'a> From<&CppObj<'a>> for Data`, so you can't do `Data::from(cpp)` here as that is `&mut CppObj`.

This creates not nice syntax of having to do `&*` to create a non-mut ref (?), so should we implement both `&CppObj` and `&mut CppObj` in the `From` ?

Reply:
Hm, maybe we will have to implement the From trait twice :thinking: 
It feels like their should be an easier solution for this, but I can't think of one atm

Reply:
I've implemented the mutable From trait by calling the immutable From trait.